### PR TITLE
QoL: freeze SB3/gym lib version, fix warning on python version

### PR DESCRIPTION
**Pull request by royerk**
My apologies for the repeated PRs. These new changes should offer a smoother experience for new users or an install to a new environment. Related issue #93 

The last version had 2 issues:
1. warning was triggered regardless of the python's version (my mistake)
2. setting `gym<0.20.0.` is not enough to ensure compatibility with `stable_baselines3`. I tested  `gym==0.17.0` which is the lowest acceptable version for the current stable baselines lib (see [here](https://github.com/DLR-RM/stable-baselines3/blob/740d61ada3360306389210a4dd8ee88ad75592db/setup.py#L76)) but it broke the `PPO.load(...)` part of the kaggle's submission (error on `Box`)

This PR introduces:
* freeze `stable_baselines3` to the current version and `gym` to `0.19.0` (tested with a new install and running a short training)
* fix the python version trigger (tested by creating a virtualenv based on python 2.7, 3.7 and 3.8)

### CODE REVIEW

File: examples/train.py [hunk 1]
```diff
@@ -173,6 +174,15 @@ def train(args):
 
 
 if __name__ == "__main__":
+    if sys.version_info < (3,7) or sys.version_info >= (3,8):
+        os.system("")
```
Comment (glmcdona) [inline examples/train.py:178]: The os.system("") isn't needed, right?

**Review by glmcdona — COMMENTED** [review]
Looks great! Maybe remove the os.system("") lines?

File: examples/train.py (hunk 1 above)
Comment (royerk) [reply examples/train.py:178]: It was recommended on a SO post to ensure compatibility with Windows. I don't have a window computer at hand to test it (see [here](https://stackoverflow.com/questions/12492810/python-how-can-i-make-the-ansi-escape-codes-to-work-also-in-windows)).

Comment (glmcdona) [reply examples/train.py:178]: Thanks, that's not what I expected!

**Pull request merged**